src/span.js

```diff
@@ -168,6 +179,19 @@ export default class Span {
     }
 
     /**
+     * Checks whether a span's sampling decision is finalized by calling setOperationName 
```

Comment:
some of this comment should be repurposed for the setOperationName function me thinks